rps.js

```diff
@@ -0,0 +1,32 @@
+let choice = prompt("Please choose one from 'rock','paper','scissors'"); 
+let array=['paper','rock','scissors'] ; 
+let random=Math.floor(Math.random() * 3); 
+let compChoice=array[random] ; 
+
+if (choice === compChoice) {
+console.log("Tie") ; } 
+else if (choice === 'paper' && compChoice ==='rock' ) {
+    console.log("You Win !") ; 
+} 
+else if (choice === 'rock'=== compChoice==='scissors') {
```

Comment:
To put multiple conditions in an if statement use **OR** operator `||` or **AND** operator `&&` to separate them.
so your if statement should looks like this: 
```else if (choice === 'rock' && compChoice==='scissors')```